src/dom/footnote.ts

```diff
@@ -52,10 +52,13 @@ export function createFootnote({
       }
     },
 
-    dismiss: () => {
+    dismiss: (onDeactivate) => {
       button.classList.add(CLASS_CHANGING)
       button.setAttribute('aria-expanded', 'false')
       button.classList.remove(CLASS_ACTIVE)
+      if (typeof onDeactivate === 'function') {
+        onDeactivate(popover)
```

Comment:
A couple of questions:

* Shouldn't we pass the `button` as well for consistency with the `onActivate` callback?
* Is this callback more useful before or after the button attributes are changed?

Reply:
> Shouldn't we pass the button as well for consistency with the onActivate callback?

I tried it but I think that I hit a strange edge case. For example when you have 2 footnotes 1 and 2, 2 is open and you click on 1. Before the activateCallback is called on 1, dismissCallback will be called for the popover2. In dismiss() in footnote.ts you don't have the information about which button has been clicked, just which button is associated to the popover being dismissed. 

> Is this callback more useful before or after the button attributes are changed?

For me the callback is more useful before the popover is removed from the DOM :) It could be called before the button attributes are changed, no big deal.

Reply:
Sorry about the delay. I was thinking about the button for the dismissed popover in this case, not the button for any other popovers that might be activating at the same time.

A use case might be making some changes to the button on activation (adding a class, etc.) while the `dismissCallback` might be used to revert those changes.